packages/client/components/SuggestedActionCard.tsx

```diff
@@ -46,14 +47,19 @@ const CancelIcon = styled(Icon)({
 
 const FloatingSealIcon = styled(Icon)({
   color: PALETTE.GRAPE_700,
+  cursor: 'pointer',
   background: PALETTE.SLATE_300,
   borderRadius: '100%',
   boxShadow: buttonShadow,
   padding: 8,
   position: 'absolute',
   fontSize: ICON_SIZE.MD36,
   top: 100,
-  userSelect: 'none'
+  userSelect: 'none',
+  opacity: 0.7,
```

Comment:
-2 this makes the problem only worse. `SuggestedActionCard` is used in multiple places and now looks clickable in all of those regardless of the icon used.

@ackernaut The issue was not well defined what the best solution would be. I don't mind having the icon repeat the primary action of a suggestion, but maybe there are some guideline against this?

Reply:
@Dschoordsch  thanks for the feedback. You can let me know what you decide